spec/requests/v0/profile/addresses_spec.rb

```diff
@@ -348,322 +332,296 @@
+        it 'matches the errors schema', :aggregate_failures do
+          address.address_pou = ''
 
-      it 'matches the errors camel-inflected schema', :aggregate_failures do
-        address.address_pou = ''
+          put('/v0/profile/addresses', params: address.to_json, headers:)
 
-        put('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response).to match_response_schema('errors')
+          expect(errors_for(response)).to include "address-pou - can't be blank"
+        end
 
-        expect(response).to have_http_status(:unprocessable_entity)
-        expect(response).to match_camelized_response_schema('errors')
-        expect(errors_for(response)).to include "address-pou - can't be blank"
-      end
-    end
+        it 'matches the errors camel-inflected schema', :aggregate_failures do
+          address.address_pou = ''
 
-    context 'when effective_end_date is included' do
-      let(:address) do
-        build(:va_profile_v3_address, effective_end_date: Time.now.utc.iso8601)
-      end
+          put('/v0/profile/addresses', params: address.to_json, headers: headers_with_camel)
 
-      it 'effective_end_date is NOT included in the request body', :aggregate_failures do
-        expect_any_instance_of(VAProfile::V2::ContactInformation::Service).to receive(:put_address) do |_, address|
-          expect(address.effective_end_date).to eq(nil)
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response).to match_camelized_response_schema('errors')
+          expect(errors_for(response)).to include "address-pou - can't be blank"
         end
-
-        put('/v0/profile/addresses', params: address.to_json, headers:)
       end
-    end
 
-    context 'when non ASCII characters are used' do
-      it 'matches the error schema' do
-        address.address_line1 = '千代田区丸の'
+      context 'when effective_end_date is included' do
+        let(:address) do
+          build(:va_profile_v3_address, effective_end_date: Time.now.utc.iso8601)
+        end
 
-        put('/v0/profile/addresses', params: address.to_json, headers:)
+        it 'effective_end_date is NOT included in the request body', :aggregate_failures do
+          expect_any_instance_of(VAProfile::V2::ContactInformation::Service).to receive(:put_address) do |_, address|
+            expect(address.effective_end_date).to eq(nil)
+          end
 
-        expect(response).to have_http_status(:unprocessable_entity)
-        expect(response).to match_response_schema('errors')
-        expect(errors_for(response)).to include 'address - must contain ASCII characters only'
+          put('/v0/profile/addresses', params: address.to_json, headers:)
+        end
       end
-    end
-  end
 
-  describe 'DELETE /v0/profile/addresses v2' do
-    before do
-      Flipper.enable(:va_v3_contact_information_service)
```

Comment:
And here as well.

Reply:
The flipper is enabled on [line 337](https://github.com/department-of-veterans-affairs/vets-api/blob/b4ed91d588872d777815af3919818d2668e4515b/spec/requests/v0/profile/addresses_spec.rb#L337)